annals-of-botany.csl

```diff
@@ -104,14 +104,25 @@
           </group>
           <text prefix=" " suffix="." macro="publisher"/>
         </if>
+        <else-if type="webpage" match="any">
+          <group suffix=". ">
+            <text macro="title" font-style="italic" prefix=" "/>
+          </group>
+          <text variable="URL" prefix=" " suffix="."/>
+          <date variable="accessed">
+            <date-part name="day" form="numeric" suffix=" "/>
+             <date-part name="month" form="short" suffix=" "/>
+             <date-part name="year" form="long"/>
+           </date>
+        </else-if>    
         <else-if type="chapter paper-conference" match="any">
           <text macro="title" prefix=" "/>
-          <group prefix=".">
+          <group prefix=" In: ">
```

Comment:
this is really helpful. Thanks for the little tips here. Exactly what I need to get better at this. Love it. Like solving puzzles.